### PR TITLE
added feeder subsystem

### PR DESCRIPTION
**Pull request by Lewis-Seiden**

### CODE REVIEW

File: src/main/java/frc/robot/subsystems/feeder/FeederSubsystem.java [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+
+import edu.wpi.first.wpilibj2.command.Command;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import org.littletonrobotics.junction.Logger;
+
+public class FeederSubsystem extends SubsystemBase {
+  final FeederIO io;
+  final FeederIOInputsAutoLogged inputs = new FeederIOInputsAutoLogged();
+
+  /** Creates a new FeederSubsystem. */
```
Comment (Awesomeplayer165) [inline src/main/java/frc/robot/subsystems/feeder/FeederSubsystem.java:15]: odd javadocs across all of these RPs?

File: src/main/java/frc/robot/subsystems/feeder/FeederIOReal.java [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+
+import com.ctre.phoenix6.BaseStatusSignal;
+import com.ctre.phoenix6.StatusSignal;
+import com.ctre.phoenix6.configs.TalonFXConfiguration;
+import com.ctre.phoenix6.controls.VoltageOut;
+import com.ctre.phoenix6.hardware.TalonFX;
+import com.ctre.phoenix6.signals.InvertedValue;
+import frc.robot.utils.components.ReversibleDigitalInput;
+
+/** Add your docs here. */
```
Comment (Awesomeplayer165) [inline src/main/java/frc/robot/subsystems/feeder/FeederIOReal.java:15]: javadocs?

**Review by Awesomeplayer165 — CHANGES_REQUESTED** [review]
small things

File: src/main/java/frc/robot/subsystems/feeder/FeederSubsystem.java (hunk 1 above)
Comment (Lewis-Seiden) [reply src/main/java/frc/robot/subsystems/feeder/FeederSubsystem.java:15]: wpilib vscode autogenerates these

File: src/main/java/frc/robot/subsystems/feeder/FeederIO.java [hunk 3]
```diff
@@ -0,0 +1,24 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package frc.robot.subsystems.feeder;
+
+import org.littletonrobotics.junction.AutoLog;
+
+public interface FeederIO {
```
Comment (Awesomeplayer165) [inline src/main/java/frc/robot/subsystems/feeder/FeederIO.java:9]: does this deliberately not have a comment?
sorry if I was not thorough enough, but suggestions last time were only duplicates wrt the whole PR.

**Review by Awesomeplayer165 — CHANGES_REQUESTED** [review]
super duper small things.

File: src/main/java/frc/robot/subsystems/feeder/FeederIO.java (hunk 3 above)
Comment (Awesomeplayer165) [reply src/main/java/frc/robot/subsystems/feeder/FeederIO.java:9]: looking through the other PRs, you have deliberately removed these comments. OK, just making sure we are consistent.

**Review by Awesomeplayer165 — APPROVED** [review]
LGTM

**Pull request merged**